### ISSUE TITLE
[Feature] Add Cash flow statement

### ISSUE THREAD

**Issue by paulavidela** [fivetran/dbt_netsuite#47]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

It would be nice to have a 'Cash Flow Statement' (like Income Statement and Balance Sheet). 
Not sure if there is information on how to know about 'Cash from operating activities', 'Cash from investing activities' and 'Cash from financing activities' 

### Describe alternatives you've considered

Writing a cash flow statement using the Income Statement and Balance Sheet

### Are you interested in contributing this feature?

- [ ] Yes.
- [X] Yes, but I will need assistance and will schedule time during your [office hours](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) for guidance.
- [ ] No.

### Anything else?

If anyone would like to work on this feature with a define spec, we would be happy to help set it up

**Comment by fivetran-joemarkiewicz** [comment]
Hi @paulavidela thanks so much for opening this Feature Request!

Adding a Cash Flow statement is something we have considered adding to this (and our other financial) packages. We have attempted to add this statement, but have found it to be a bit more difficult to implement dynamically for all use cases within our packages. While that is the case, I would be interested in adding this as a possible feature for us to explore in a future quarter. As such, I will keep this FR open until we pick this up. 

In the meantime, if anyone has any helpful information to add, or just wants to upvote this FR, please feel free to do so. Any and all information will help our team once we decide to pick this up!

**Comment by ropalloni** [comment]
This would be a really useful model!

**Comment by MartinaLucente** [comment]
This feature is definitely useful!

**Comment by stolomeo911** [comment]
Upvote!

**Comment by rpalloni** [comment]
Upvoting!!!

**Comment by ropalloni** [comment]
Any news on this? Just to be aware of.

**Comment by fivetran-joemarkiewicz** [comment]
Hi @rpalloni we are currently working on this enhancement to our QuickBooks package. Once we are able to successfully implement the Cash Flow statement within QuickBooks, we will look to bringing this enhancements into the fold for Netsuite.

**Comment by ropalloni** [comment]
Ok thanks for the update. Will it be available also for NetSuite2?

**Comment by fivetran-joemarkiewicz** [comment]
Yeah! That will be our intention to have it available for Netsuite2 as well as Netsuite versions of the package.

**Comment by reubster** [comment]
upvote

**Comment by bmayer99** [comment]
might it be worth starting with trial balances by period? or the union of the IS and BS reports?
I have noticed the BS does not have Subsidiary level information while the IS does.

**Comment by fivetran-catfritz** [comment]
Hi @bmayer99 @reubster @rpalloni @paulavidela @stolomeo911 @MartinaLucente,

Good news—I’ve taken on building the Cash Flow Report! With that in mind, I could use your insights on the following questions. Any input is valuable, so no need to answer everything.

- What key cash flow metrics do you track most often?
- How do you analyze trends, and/or what challenges do you face?
- Any specific formats or visualizations that would make this more actionable?
- Any other feedback or suggestions?

Thank you!
